node.lua

```diff
@@ -79,6 +90,12 @@ minetest.register_node("digibuilder:digibuilder", {
 		return inv:is_empty("main") and not minetest.is_protected(pos, name)
 	end,
 
+	after_dig_node = function(pos)
+		if has_pipeworks then
+			pipeworks.after_dig(pos)
+		end
+	end,
```

Comment:
I'd do it like this, because then `after_dig_node` remains nil if there is no pipeworks. (no function called)
```suggestion
	after_dig_node = has_pipeworks and pipeworks.after_dig or nil,
```

Reply:
good point